src/Clide/Commands/CommandManager.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.Composition;
+using System.ComponentModel.Design;
+using System.Linq;
+
+namespace Clide.Commands
+{
+	[Export(typeof(ICommandManager))]
+	class CommandManager : ICommandManager
+	{
+		readonly Lazy<IMenuCommandService> menuCommandService;
+		readonly Lazy<IServiceLocator> serviceLocator;
+
+		[ImportingConstructor]
+		public CommandManager(
+			[Import(ContractNames.Interop.IMenuCommandService)] Lazy<IMenuCommandService> menuCommandService,
```

Comment:
I don't think we need interop contract names for any of our stuff in Clide v3. Since in VS the latest Clide will win, that will mean the workload-provided one. There cannot be two. And since Clide v2 didn't export to MEF (or we *do* use contract names in that case), the v3 can remain cleaner.